spring-boot-security-taglibs/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+#jsp config
+spring.mvc.view.prefix: /WEB-INF/views/
+spring.mvc.view.suffix: .jsp
+spring.http.encoding.charset=UTF-8
```

Comment:
If the encoding here is not strictly necessary, let's add a comment here that separates these from what is necessary for supporting JSPs. Something like:

```yaml
# The encoding settings here are not strictly necessary 
# for adding spring security tags to your JSPs, but they 
# are a good practice for secure and resilient JSPs themselves.
```

The reason for this is because many readers are probably already using Spring Security in a way that they like, but they need to know how to add JSP support. If they already have encoding settings that they are satisfied with, we don't want to miscommunicate that these are required (e.g. that they need to change their existing settings).

I'm sure you've been there before. You are trying something new, following a tutorial, something doesn't work, and so you change your code to be exactly like theirs. Well, let's help them out by differentiating what is necessary and what isn't.

And if they are necessary, then go ahead and ignore all of that. :)